packages/main/src/ListItem.ts

```diff
@@ -257,6 +257,18 @@ abstract class ListItem extends ListItemBase {
 	@slot()
 	deleteButton!: Array<HTMLElement>;
 
+	/**
+	 * Defines the icon, displayed in the beginning or end of the list item.
```

Comment:
Minor:
Maybe also add a comment that if a customIcon is set, it will be rendered in place of the standard icon defined in the "icon" property